source/intro.tex

```diff
@@ -60,6 +60,9 @@
 \item ISO 80000-2:2009, \doccite{Quantities and units ---
 Part 2: Mathematical signs and symbols
 to be used in the natural sciences and technology}
+\item Unicode Standard Annex, UAX \#29, \doccite{Unicode Text Segmentations},
```

Comment:
I've made a further suggestion (attached to the discussion on the previous version of this): https://github.com/cplusplus/draft/pull/3753#discussion_r383037073. Also, the title should have "Segmentation" (that is, without the "s").

Reply:
thanks!